e2e/process-services/process-attachmentList-actionMenu.e2e.ts

```diff
@@ -140,7 +140,7 @@ describe('Attachment list action menu for processes', () => {
         attachmentListPage.clickAttachFileButton(pngFile.location);
 
         processDetailsPage.clickCancelProcessButton();
-        browser.sleep(10000);
+        browser.sleep(3000);
```

Comment:
do we really need sleep here? I thought we've been using "wait for element" or something like that